File: 110algoritmos/exercicios008/ex008-Go/ex008.go
package main

import "fmt"

func main() {
	var metros float32

	fmt.Print("Digite uma distancia em metros: ")
	fmt.Scan(&metros)

	cent := metros * 100
	mili := metros * 1000

	fmt.Printf("%v metros sao %v centimetros e %v milimetros.", metros, cent, mili)

}

File: 110algoritmos/exercicios008/ex008-Csharp/ex008.cs
using System;

class Program {

    static void main() {

        float metros = 0.0;
        float cent, mili;

        Console.Write("Digite um valor em metros: ");
        metros = float.Parse(Console.ReadLine());

        cent = metros * 100;
        mili = metros * 1000;

        Console.WriteLine(metros + " metros sao " + cent + " centimentos e " + mili + " milimetros.");

    }

}